### PR TITLE
completed sql-select - Jonathan Chiang

### PR DESCRIPTION
**Pull request by J0N-C**
Apologies ahead of time for the long series of screenshots to come.

1. sql-select-all-actors
2. sql-select-actor-names
3. sql-select-actor-forty
4. sql-select-ten-actor-first-names
5. sql-select-two-kilmers
6. sql-select-all-categories
7. sql-select-all-customers
8. sql-select-inactive-customers
9. sql-select-five-customer-emails
10. sql-select-second-store-customers
11. sql-select-all-languages
12. sql-select-all-films
13. sql-select-twenty-films-of-06
14. sql-select-r-rated-films
15. sql-select-expensive-films
16. sql-select-cheap-to-rent
17. sql-select-west-bengali-postal-codes

### CODE REVIEW

File: sql-select/actor-forty.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+select "firstName",
+  "lastName"
+  from "actors"
+  where "actorId" = '40';
```
Comment (TimelikeClosure) [inline sql-select/actor-forty.sql]: Generally, if a value is a non-string-type, you shouldn't need to wrap it in single-quotes.

File: sql-select/twenty-films-of-06.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+select  "filmId",
```
Comment (TimelikeClosure) [inline sql-select/twenty-films-of-06.sql]: `"filmId"` isn't in the list of fields that should be selected.

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by J0N-C** [comment]
sql-select-actor-forty-rev
sql-select-twenty-films-of-06-rev

**Review by TimelikeClosure — APPROVED** [review]

File: sql-select/actor-forty.sql (hunk 1 above)
Comment (J0N-C) [reply sql-select/actor-forty.sql]: Would numbers be the only non string type in sql databases or are there other data types that can be stored here?

**Pull request merged**